.eslintrc.js

```diff
@@ -94,5 +94,9 @@ module.exports = {
     'react-native/no-unused-styles': ERROR,
     'react-native/no-inline-styles': WARNING,
     'flowtype/require-valid-file-annotation': [ERROR, 'always'],
+    'flowtype/newline-after-flow-annotation': [
+      2,
```

Comment:
`ERROR` to be more obvious, please... :)

Reply:
Sorry, copy/paste `rookie-mistake`